charts/wa-task-management-api/values.preview.template.yaml

```diff
@@ -1,5 +1,8 @@
 java:
   environment:
-  # Don't modify below here
+    CAMUNDA_URL: "http://camunda-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal/engine-rest"
```

Comment:
no need to duplicate values here if they are defined in the values.yaml already.